src/tool_cb_hdr.c

```diff
@@ -363,6 +364,7 @@ void write_linked_location(CURL *curl, const char *location, size_t loclen,
   while(llen && *loc == ' ') {
```

Comment:
It turns out that HTTP allows a tab character here as whitespace as well, so that should also be skipped.